src/test/java/com/google/api/codegen/testdata/java_mock_service_impl_library.baseline

```diff
@@ -297,8 +284,7 @@ public class MockLibraryServiceImpl implements LibraryService  {
   public void updateBookIndex(UpdateBookIndexRequest request,
       StreamObserver<Empty> responseObserver) {
     if (responses != null && !responses.isEmpty()) {
-      Empty response = (Empty) responses.get(0);
-      responses.remove(0);
+      Empty response = (Empty) responses.poll();
```

Comment:
Yes I agree that we should break hard and fast especially in a test code. But correct me if I am wrong, here the mock service is shared by all the tests so if we throw an exception (and interrupt the execution) here, it would fail all the following tests. Ideally we would like only fail the test that caused the issue, which requires the mock service to continue executing.